### ISSUE TITLE
Allow for fitting the susceptible population

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/EpiNow2#408]
It would be useful to allow for fitting the susceptible population (currently passed as pop). This would require enabling a mode where the susceptibility correction was applied to observed data as well as just to beyond the forecast horizon.

**Comment by sbfnk** [comment]
> enabling a mode where the susceptibility correction was applied to observed data as well as just to beyond the forecast horizon.

This, I think, would be a sensible change anyway.

**Comment by seabbs** [comment]
Yes agree. A complexity is returning the resulting Rt (as can currently only return one thing from a function) and if its worth communicating both adjusted and unadjusted Rt

**Comment by seabbs** [comment]
note you can already achieve this in practice if you moved the pop adjustment into the estimation window and used a ascertainment model but that isn't very clear and is a bit of a hack.

**Comment by seabbs** [comment]
With the new interface froom @sbfnk this would now be very easy to do
